### PR TITLE
A11y 48 remove duplicate help site button link

### PR DESCRIPTION
**Pull request by mgc1194**
As a part of our Accessibility effort, we are trying to make our pages more screen reader and tab navigation friendly, and compliant with WCAG standards.
A screen reader  on [How to help](https://code.org/help) reads out each button twice, once for the link and once for the button. 

This PR removes the redundancy, turning buttons with links into links with a button-like style. 
For this purpose, a `link-button` style has been added to the common Style Sheet, and to class attribute of updated links. This approach represents a better overall design as new styles are available across other pages, rather than specified in each markdown/html/haml file.

## Links

- jira ticket: [A11Y-48](https://codedotorg.atlassian.net/browse/A11Y-48)

## Testing story

Tested manually that all links redirect correctly.

Production vs Development comparative image.
image
 
The following video captures the behavior of the previous buttons vs new links using tab navigation and a screen reader.
Note that buttons are cover twice when navigating the page with keyboard (tab navigation)  
https://user-images.githubusercontent.com/66776217/216675344-8dbdc0f1-8aad-43b9-9ac3-149567fc061b.mov

## Deployment strategy
This PR will result in failing eyes test as there is a slight change in the buttons position.

## Follow-up work
Updating any other button with links in other parts of pegasus code (more specifically in markdowns) 
The following list contains some pages with button and links that need to be updated.
- https://code.org/donate/amazonsmile
- https://code.org/promote
- https://hourofcode.com/us/es/how-to
- https://code.org/international

## PR Checklist:

- [x] Tests provide adequate coverage
- [x] Privacy and Security impacts have been assessed
- [x] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [x] Relevant documentation has been added or updated
- [x] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately
- [x] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/common.css [hunk 1]
```diff
@@ -403,6 +403,17 @@ button {
...
+a.link-button {
+  display: inline-block;
+  color: white;
+  background-color: #ffa400;
+  border-radius: 4px;
+  padding: 6px 12px;
+  font-size: 14px;
+  font-family: 'Gotham 4r', sans-serif;
+  text-decoration: none;
+}
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/common.css:415]: @mgc1194 just updated the styles 🎉 

Here's how it looks:

Screenshot 2023-02-01 at 2 32 05 PM

**Review by epeach — APPROVED** [review]
Love it! Links are links! And thank you for adding that styling to common so we can hopefully reuse as needed!

**Comment by breville** [comment]
Is this the approach that we'll scale across our entire site?

**Comment by mgc1194** [comment]
I am under the impression that this is best practice. Buttons tags should not have link tags inside for accessibility reasons. And I believe it is more appropriate to style a link to look like a button. The design team probably can explain this better.
And answering to your question @breville, the accessibility working group discussed the next steps to update other existing buttons with links inside to this approach.

**Comment by mgc1194** [comment]
I am under the impression that this is best practice. Buttons tags should not have link tags inside for accessibility reasons. And I believe it is more appropriate to style a link to look like a button. The design team probably can explain this better.
And answering to your question @breville, the accessibility working group discussed the next steps to update other existing buttons with links inside to this approach.

**Pull request merged**